--- CMakeLists.txt	2020-01-28 22:15:30.852948000 +0100
+++ CMakeLists.txt	2020-01-28 22:19:11.708740942 +0100
@@ -393,6 +393,9 @@
 elseif(GNU)
 	add_definitions(-DGNU=1)
 	message(STATUS "DETECTED SYSTEM: GNU;  GNU=1" )
+elseif(ANDROID)
+	add_definitions(-DANDROID=1)
+	message(STATUS "DETECTED SYSTEM: ANDROID; ANDROID=1")
 else()
 	message(FATAL_ERROR "Unsupported system: ${CMAKE_SYSTEM_NAME}")
 endif()
