From d303b9733a7446ccbefbfced73b79d2a70cea22e Mon Sep 17 00:00:00 2001
From: ThibaultBee <thibault@api.video>
Date: Thu, 17 Feb 2022 14:34:40 +0100
Subject: [PATCH] android-x86 target: Add -latomic for clang

---
 Configurations/15-android.conf | 1 +
 1 file changed, 1 insertion(+)

diff --git a/Configurations/15-android.conf b/Configurations/15-android.conf
index 0b6b6d6832..41ad9223e0 100644
--- a/Configurations/15-android.conf
+++ b/Configurations/15-android.conf
@@ -261,6 +261,7 @@ my %targets = (
         bn_ops           => add("RC4_INT"),
         asm_arch         => 'x86',
         perlasm_scheme   => "android",
+        ex_libs          => add(threads("-latomic")),
     },
     "android-x86_64" => {
         inherit_from     => [ "android" ],
-- 
2.34.1

